pkg/operator/operatorclient/interfaces.go

```diff
@@ -5,4 +5,5 @@ const (
 	GlobalMachineSpecifiedConfigNamespace = "openshift-config-managed"
 	OperatorNamespace                     = "openshift-apiserver-operator"
 	TargetNamespace                       = "openshift-apiserver"
+	ImageRegistryNamespace                = "openshift-image-registry"
```

Comment:
i hate these consts, they make grepping code nightmare, can we just inline them?

Reply:
+1

Reply:
Removed.